### ISSUE TITLE
skipExistingFileName ingored

### ISSUE THREAD

**Issue by bbo2adwuff** [ibrod83/nodejs-file-downloader#20]
Even when setting `skipExistingFileName` file gets downloaded if file with same name present in `directory`.

## How to reproduce

1. Run code below
2. Check if file `./downloads/200MB.zip` exists.
3. Run code second time
4. `./downloads/200MB_2.zip` exists.

```javascript
const Downloader = require('nodejs-file-downloader');

(async () => {//Wrapping the code with an async function, just for the sake of example.

    const downloader = new Downloader({
      url: 'http://212.183.159.230/200MB.zip',//If the file name already exists, a new file with the name 200MB1.zip is created.     
      directory: "./downloads",//This folder will be created, if it doesn't exist.         
      skipExistingFileName: true,      
    })
    try {
      await downloader.download();//Downloader.download() returns a promise.

      console.log('All done');
    } catch (error) {//IMPORTANT: Handle a possible error. An error is thrown in case of network errors, or status codes of 400 and above.
      //Note that if the maxAttempts is set to higher than 1, the error is thrown only if all attempts fail.
      console.log('Download failed',error)
    }


})();    
```

## Expected behavior 

`./downloads/200MB_2.zip` shouldn't exist.

## Version
nodejs-file-downloader 4.7.2
node --version v16.6.0
Archlinux 5.13.7

**Comment by bbo2adwuff** [comment]
Ah I just gave it a try with:

```javascript
        cloneFiles: false,
        skipExistingFileName: true,
```

Then it works. Okay I didn't expect that.

I'll just leave this issue open and it's up to you to close it. IMHO get `skipExistingFileName` working without `cloneFiles` feels more intuitive or at least mentioning it in the [readme.md](https://github.com/ibrod83/nodejs-file-downloader/blob/master/readme.md) would help.

**Comment by ibrod83** [comment]
Thank you for your input. I'll look into it

**Comment by ibrod83** [comment]
fixed in 4.7.4
